### PR TITLE
fix composition and docs for ui/docs, ui/community-docs

### LINKED ISSUE #97: fix composition for ui/docs and add test

### PR DESCRIPTION
**Pull request by JoshK2**
closed #97

### CODE REVIEW

File: docs/ui/community-docs/community-docs.docs.mdx [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
+description: 'Community docs component.'
 labels: ['docs', 'page', 'routes']
 ---
 
 import { CommunityDocs } from './community-docs';
 
-## React Component for rendering text
-
-A basic div that renders some text
+Community docs component use the Docs component to render the whole docs page.
```
Comment (giteden) [inline docs/ui/community-docs/community-docs.docs.mdx:8]: The 'Community Docs' component uses the 'Docs' component to render the doc (MDX) page.

File: docs/ui/docs/docs.docs.mdx [hunk 2]
```diff
@@ -1,24 +1,70 @@
...
+description: 'Main docs component.'
+labels: ['docs', 'page', 'routes']
 ---
 
 import { Docs } from './docs';
 
-## React Component for rendering text
-
-A basic div that renders some text
+Docs component that renders the docs page with the Sidebar on the left side and the Content on the right.
```
Comment (giteden) [inline docs/ui/docs/docs.docs.mdx:8]: The 'Docs' component renders the docs page (MDX) with a sidebar to its left.

File: docs/ui/pages/guides/guides.docs.mdx [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-labels: ['label1', 'label2', 'label3']
+labels: ['learn', 'guides', 'docs']
 ---
 
 import { Guides } from './guides';
 
-## React Component for rendering text
-
-A basic div that renders some text
+Guides component use the Docs component to render the whole guides page.
```
Comment (giteden) [inline docs/ui/pages/guides/guides.docs.mdx:8]: The 'Guides' component uses the Docs component to render  'guide' pages.

**Review by giteden — CHANGES_REQUESTED** [review]

File: docs/ui/community-docs/community-docs.docs.mdx (hunk 1 above)
Comment (JoshK2) [reply docs/ui/community-docs/community-docs.docs.mdx:8]: done

File: docs/ui/docs/docs.docs.mdx (hunk 2 above)
Comment (JoshK2) [reply docs/ui/docs/docs.docs.mdx:8]: done

File: docs/ui/pages/guides/guides.docs.mdx (hunk 3 above)
Comment (JoshK2) [reply docs/ui/pages/guides/guides.docs.mdx:8]: done

**Review by giteden — APPROVED** [review]

**Pull request merged**